contracts/VickreyAuction.sol

```diff
@@ -162,12 +158,10 @@ contract VickreyAuction is Auction {
         emit Withdrawn(auctionId, withdrawAmount);
     }
 
-    function claim(uint256 auctionId) external exists(auctionId) {
+    function claim(uint256 auctionId) external exists(auctionId) onlyAfterDeadline(auctions[auctionId].bidRevealEnd) notClaimed(auctions[auctionId].isClaimed) {
         AuctionData storage auction = auctions[auctionId];
-        require(block.timestamp > auction.bidRevealEnd, 'Reveal period has not ended yet');
-        require(!auction.isClaimed, 'Auction had been settled');
         auction.isClaimed = true;
-        if(auction.winner!=auction.auctioneer) sendFunds(false, auction.biddingToken, auction.winner, bids[auctionId][auction.winner] - auction.winningBid);
+        if(bids[auctionId][auction.winner] != auction.winningBid) sendFunds(false, auction.biddingToken, auction.winner, bids[auctionId][auction.winner] - auction.winningBid);
```

Comment:
Could you explain this to me?

Reply:
Ah, OK! Please add the following comment to the right side then: 

`// The conditional prevents sending a refund of 0`.